x-pack/packages/ml/aiops_common/constants.ts

```diff
@@ -16,9 +16,15 @@ export const AIOPS_API_ENDPOINT = {
   CATEGORIZATION_FIELD_VALIDATION: '/internal/aiops/categorization_field_validation',
 } as const;
 
-export const AIOPS_TELEMETRY_ID = {
-  AIOPS_DEFAULT_SOURCE: 'ml_aiops_labs',
-  AIOPS_ANALYSIS_RUN_ORIGIN: 'aiops-analysis-run-origin',
-} as const;
+/**
+ * Used for telemetry purposes to track the origin of the analysis run.
+ */
+export const AIOPS_ANALYSIS_RUN_ORIGIN = 'aiops-analysis-run-origin';
 
-export const EMBEDDABLE_ORIGIN = 'embeddable';
+export const AIOPS_EMBEDDABLE_ORIGIN = {
+  CASES: 'cases',
+  DASHBOARD: 'dashboard',
+  DEFAULT: 'embeddable',
+  DISCOVER: 'discover',
+  ML_AIOPS_LABS: 'ml_aiops_labs',
+} as const;
```

Comment:
Originally I wanted to use this to enforce this as a union type when adding `embeddingOrigin` on the context value. However, this turned out to be tricky for example in Discover where we pass on another value from Discover that's just typed as `string`. I left this in to be reusable within our own code even when the type is not enforced, it might help with tracking of what kinds of origins we expect to be passed on.